gammapy/time/plot.py

```diff
@@ -2,31 +2,126 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-__all__ = ['plot_time_difference_distribution',
+from ..datasets import fetch_fermi_catalog
+
+__all__ = ['plot_light_curve',
            ]
 
 
-def plot_time_difference_distribution(time, ax=None):
-    """Plot event time difference distribution.
+def plot_light_curve(name_3FGL, fermi_met_start = 2.3933e8, fermi_met_end = 3.65467550e8):
+    """Plot flux as a function of time for a fermi 3FGL object.
 
     Parameters
     ----------
-    time : `~astropy.time.Time`
-        Event times (must be sorted)
-    ax : `~matplotlib.axes.Axes` or None
-        Axes
-
-    Returns
-    -------
-    ax : `~matplotlib.axes.Axes`
-        Axes
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `int`
```

Comment:
I don't know if you've seen it, yesterday I wrote some notes on [time handling in Gammapy](https://gammapy.readthedocs.org/en/latest/time/index.html#time-handling-in-gammapy)

Any of these would be valid choices I think:
* Don't take start and end time as input, always plot the full light curve.
* Take two `astropy.time.Time` objects as input called `start_time` and `end_time`
* Take one `astropy.time.Time` object as input with length 2 called `time_range`